GitUI/BranchTreePanel/RepoObjectsTree.Nodes.Branches.cs

```diff
@@ -237,12 +240,14 @@ private void FillBranchTree(IEnumerable<string> branches, CancellationToken toke
 
                 #endregion
 
+                var aheadBehindData = _aheadBehindDataProvider.GetData();
```

Comment:
_aheadBehindDataProvider can be null (but I do not see a crash)

Reply:
You're right, will make it ```?.GetData();```